package.json

```diff
@@ -38,7 +38,7 @@
     "shallowequal": "^1.0.2"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-rc"
+    "react": "^0.14.0 || ^15.0.0-0 || ^17.0.2"
```

Comment:
can we change this to `"react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-rc || ^17.0.2"`?